### DISCUSSION TITLE
Invisible SPS?

### DISCUSSION THREAD

**Discussion by maxq519** [orbiternassp/NASSP#748] [Ideas]
This is just an idea.  In reality, it seems the exhaust from the SPS is not visible.  I was thinking that maybe this might be a good feature to have in NASSP?  Probably something the user can select, it might seem kinda odd to not have any visual feedback that the engine is running…

Here is my reference:
[https://www.hq.nasa.gov/alsj/a14/AS14_CM.PDF](url)
Page 187.
“ You have a mighty invisible engine, Stu. It
didn't have anything we could see at all from here.”

Actually, this is probably the case for the DPS/APS as well, for the APS at least just watch a video of the Apollo 17 ascent.

Thoughts/Comments?

**Comment by n7275** [comment]
You can definitely see some parts of thrusters firing: https://youtu.be/EeeeTVr4Lyg?t=522

The whole plume probably isn't visible, but the flow through the thruster/SPS/APS/DPS throat would be very dense and bright. You might be able to do this with an emissive texture, but replacing the plumes with something volumetric and barely visible is probably going to be hard to make it not look tacky.

**Comment by maxq519** [comment]
What about making the exhaust texture shorter?  That way you could see the glow when looking down the engine bell, but not when looking at the engine from the side.  As near the end of the video when the LM pitches down and yaws, we can see the yaw maneuver start and stop, but no thrusters.  Seems the bright part is right near the nozzle throat only, which makes sense because density (and temperature) would drop off real fast as the exhaust gasses are expanded in the nozzle.
